lib/order.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'send_sms'
+require_relative 'dishes'
+
+class Order
+
+  attr_reader :selection, :dishes, :sms
+
+  def initialize(dishes = Dishes.new, sms = SendSMS.new)
+    @dishes = dishes
+    @sms = sms
+  end
+
+  def view_dishes
+    dishes.view
+  end
+
+  def create(selection)
+    @selection = selection
+  end
```

Comment:
Maybe a hash could have been used here? Or was there a reason for otherwise?

Reply:
Using a hash would then make it easier to manage separate items/counts and a delete function could even be implemented later?